aether-entity-extraction-module/setup.cfg

```diff
@@ -31,7 +31,9 @@ description-file = README.md
 
 [tool:pytest]
 python_files = tests/test*.py
-addopts = --maxfail=100 -s --capture=no
+addopts =
+    --maxfail=100
+    -s
 
 [coverage:run]
 branch = true
```

Comment:
you could add here the option `parallel =  true` and remove it from the `entrypoint.sh` file